mantidproject/mantid
# Deprecate Old SANS GUI

**Description of work.**
This PR changes the name of the two ISIS SANS GUIs. The old GUI is now `ISIS SANS (Deprecated)` and the "new" GUI is `ISIS SANS`.

**Report to:** nobody

**To test:**
1. MantidPlot -> Interfaces -> SANS. There should be, among others, `ISIS SANS (Deprecated)` and `ISIS SANS`. Check that both interfaces open
2. Workbench -> Interfaces -> SANS. There should be an `ISIS SANS` interface, but no deprecated one. Check that it opens.

Fixes #24854 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
